src/client/filtering/FuncOperations.js

```diff
@@ -126,6 +126,19 @@ export const setOperation = (op, value) => {
   }
 };
 
+export const setContainsOperation = (op, value) => {
```

Comment:
Added this because the existing set operations didn't fit well. setElementOperation didn't work even though legality filter is similar and setOperation isn't doing subsets in general.